src/components/offering/sub-title.tsx

```diff
@@ -0,0 +1,13 @@
+import { colors } from '@hedviginsurance/brand'
+import styled from 'react-emotion'
+
+export const SubTitle = styled('p')({
```

Comment:
Unsure if `p` is the best use case for a sub title? Maybe its better to use something like `div` or a lower level of `hN`?

Reply:
+1 - try to use the correct semantic tags, it's good for everyone!

Reply:
Good point